docker-compose.yml

```diff
@@ -0,0 +1,20 @@
+networks:
+  esp_wireguard_network:
+    driver: bridge
+    ipam:
+      config:
+        - subnet: 172.18.0.0/16
+
+services:
+  esp_wireguard_examples_demo:
```

Comment:
In the README you showed this network:
```
               wireguard_peer1
               /
              /
esp_wireguard
              \
               \
                wireguard_peer2
```

I was under the impression, that wireguard_peer1 and wireguard_peer2 would get their own containers. Should they, or am I missing something?

Reply:
Oh, somehow with my latest [commit](https://github.com/XmasHackathon2023/esp_wireguard/compare/78395746197380d5cffdf6b208da8dd4cd515fd7..a8980f0a7ad85e7e93463419f4ce6c29f1bdbf45) I got it removed. Will fix it

Reply:
Pushed